source/_data/glossary.yml

```
@@ -200,20 +200,15 @@
   link: /installation/#advanced-installation-methods
 - term: Home Assistant Core
   definition: >-
-    Home Assistant Core is a Python program. It can be run on various operating
-    systems and is the basis for Home Assistant. When people are talking about
-    Home Assistant Core they usually refer to a standalone installation method
-    that can be installed using a Virtual Environment or Docker. Home Assistant
-    Core does not use the Home Assistant Supervisor.
+    Home Assistant Core is the Python program at the heart of Home Assistant. It is part of all of the four installation types and can be run on various operating systems. Home Assistant Core is also the name of an installation method. It can be installed standalone (without Home Assistant Supervisor), using a Virtual Environment or Docker.
   link: /installation/#advanced-installation-methods
   excerpt: >
     Home Assistant Core is the heart of Home Assistant itself. It is a Python
     program that powers every installation type, but can be installed standalone.
 - term: Home Assistant Supervised
   definition: >-
-    Home Assistant Supervised is a full UI managed home automation ecosystem that
-    runs Home Assistant, the Home Assistant Supervisor and add-ons. It comes
-    pre-installed on Home Assistant OS, but can be installed on any Linux system.
+    Home Assistant Supervised is a full UI managed home automation ecosystem that runs the Home Assistant Core program, the Home Assistant Supervisor and add-ons. It comes
+    pre-installed on Home Assistant OS, but can be installed standalone on any Linux system. 
```

The officially supported platforms is Debian, so I'd advertise that only in this case:

```suggestion
    pre-installed on Home Assistant OS, but can be installed standalone on Debian Linux systems.
```